cli/internal/scope/filter/filter.go

```diff
@@ -210,7 +210,7 @@ func (r *Resolver) filterNodesWithSelector(selector *TargetSelector) (util.Set,
 					} else if matches {
 						entryPackages.Add(pkgName)
 					}
-				} else if pkg, ok := r.PackageInfos[pkgName]; !ok {
+				} else if pkg, ok := r.WorkspaceInfos[pkgName.(string)]; !ok {
```

Comment:
Not that it especially matters, but I'd like to the type assertion earlier in the loop or at least on a different line from the lookup. Since both type assertions and map lookups return a tuple with a bool indicating success it could be easy to misread the `ok` as coming from `_, ok := pkgName.(string)`

Reply:
Hmm. cascading effect of changing types. `pkgName` is a coming from a `util.Set`, which is 
```
type Set map[interface{}]interface{}
```

Might be useful to have a `StringSet` or `PackageNamesSet` type instead..., but that could cascade also. 

Update this piece for now